README.md

```diff
@@ -19,6 +19,7 @@ cluster on top of bare metal infrastructure using Metal3.
 |---------------|---------------------|---------------|
 | v1alpha4      | v1alpha3            | v0.4.X        |
 | v1alpha5      | v1alpha4            | v0.5.X        |
+| v1beta1       | v1beta1             | v1.0.X        |
```

Comment:
Do we have to follow the releases scheme too?

Reply:
I thought we follow it, at least I am taking reference from other providers, are you suggesting it to be v0.6,x?

Reply:
not suggesting, but rather opening a question to discuss if we need it be this way or the other

Reply:
As discussed offline, we will stick with the versioning scheme noted in the CAPI compatibility table.